k: 1.5
u_min: -640.0
u_max: 640.0
v_min: -400.0
v_max: 400.0
quadCorners3d:
  rows: 4
  cols: 3
  data: [-0.15, -0.15, 0,
          0.15, -0.15, 0,
          0.15,  0.15, 0,
         -0.15,  0.15, 0]